fiftyone/utils/unlimited_ocr.py

```
@@ -0,0 +1,255 @@
+    Args:
+        config: an :class:`UnlimitedOCRModelConfig`
+    """
+
+    def __init__(self, config):
+        self._tokenizer = None
+        super().__init__(config)
+
+    def _download_model(self, config):
+        pass  # transformers downloads the model on first load
+
+    def _load_model(self, config):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_load_model`

(ANN202)